flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/SourceOperator.java

```diff
@@ -167,6 +164,9 @@
     private InternalSourceReaderMetricGroup sourceMetricGroup;
 
     @Nullable private WatermarkTrackingOutput<OUT> watermarkTrackingOutput;
+    private final boolean alignmentEnabled;
+    private final long alignmentUpdateInterval;
```

Comment:
nit: for the sake of consistence with `SourceCoordinator` and completeness, I would keep the `WatermarkAlignementParams` here. You could re-use strategy -> params conversion code in both places.